package.json

```diff
@@ -47,7 +48,7 @@
     "husky": "^1.0.0-rc.13",
     "lint-staged": "^7.2.2",
     "mocha": "^5.2.0",
-    "nock": "^9.3.3",
+    "nock": "^13.0.4",
```

Comment:
nock was causing tests to fail, as it wasn't delaying connection on later version of node. __fun__